Source/ACE.Server/WorldObjects/Player_Tracking.cs

```diff
@@ -63,7 +63,8 @@ public void TrackObject(WorldObject worldObject)
             if (worldObject is Creature creature)
             {
                 foreach (var wieldedItem in creature.EquippedObjects.Values)
-                    TrackEquippedObject(creature, wieldedItem);
+                    if (IsInChildLocation(wieldedItem))
```

Comment:
I don't think this will work, ammo is not a child object

Reply:
see new commit 6fc802b19f8dedcc9b54ecb0675f3c536ead84e2